archive/bagger/src/identifiers.py

```diff
@@ -9,7 +9,7 @@ def normalise_b_number(b_str):
     if len(b_str) != 8:
         raise ValueError("Not a valid b number")
 
-    if b_str == "19974760":
-        raise ValueError("Will not process Chemist and Druggist right now")
+    # if b_str == "19974760":
+    #     raise ValueError("Will not process Chemist and Druggist right now")
```

Comment:
This is going to conflict with the enable c&d branch no?

Reply:
accidentally added that file, have reverted